### DISCUSSION TITLE
[Feature] Can we get the container list sidebar as a collapsible sidebar?

### DISCUSSION THREAD

**Discussion by arpanghosh8453** [louislam/dockge#355] [Feature Request]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

It will be great to collapse the sidebar for better navigation and logs reading on smaller screens. 

### 🐻 Dockge Version

1

### 💻 Operating System and Arch

Debian 12

### 🌐 Browser

Chrome

### 🐋 Docker Version

24.0.7

**Comment by docker-master** [comment]
If you enlarge the compose code, it will be better like portainer. It is convenient to add comments and there will be no line breaks.

**Comment by quantumfrost** [comment]
I agree, sometimes I have to work on a laptop with two windows side-by-side. That currently makes dockge unusable

**Comment by motoridersd** [comment]
Would be nice to be able to collapse/expand all sections.

Collapse the container list on the left, expand/collapse the Terminal window, expand/collapse the compose window, even if it means opening in a new window.

**Comment by Jekotia** [comment]
I would argue that dockge is currently borderline unusable even in fullscreen, when using a 13" laptop (Framework 13 in my case).

![Screenshot 2025-04-11 125208](https://github.com/user-attachments/assets/eee337dd-9342-4d91-bb20-0b802d594b50)

The display of the compose file is absolutely terrible. Lines are wrapped and given new line numbers, because a full third of the screen (!) is being given to a column that should be a sidebar. It is equally bad when clicking the edit button (colours change and the text area becomes editible).

**Comment by BenRoe** [comment]
Please add this feature.
